### DISCUSSION TITLE
Potential bug / undocumented deprecation of NullBooleanField?

### DISCUSSION THREAD

**Discussion by martin-thoma** [encode/django-rest-framework#8667] [Question & Answer]
See https://stackoverflow.com/q/73823743/562769

**Comment by mattiasostmar** [comment]
According to this, it has been fixed https://github.com/heartexlabs/label-studio/issues/2996#issuecomment-1258839504

**Reply by mattiasostmar** [reply]
This open issue seems to be related https://github.com/encode/django-rest-framework/pull/7485
